{"uuid": "d6547c2b-cb77-41f0-b722-b6c43c9cb77b", "children": ["1f9085b0-58df-4ab1-a85f-7355d0a2bf6f", "819d09a8-ad71-4f96-8f7c-329919d67920", "4aab7d29-6d7a-4b50-adf5-92e3ab42aa6d", "3af60885-1d22-409c-968f-4a37860a022a", "4b7bd123-e18e-45fc-a8f0-9691d8aa7c7b", "a3be454d-25f6-4f3e-acd1-80d435552517"], "befores": [{"name": "browser", "status": "broken", "statusDetails": {"message": "playwright._impl._api_types.Error: It looks like you are using Playwright Sync API inside the asyncio loop.\nPlease use the Async API instead.\n", "trace": "  File \"/home/kamaz/PycharmProjects/FinalProject/venv/lib/python3.11/site-packages/pluggy/_callers.py\", line 77, in _multicall\n    res = hook_impl.function(*args)\n          ^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/home/kamaz/PycharmProjects/FinalProject/venv/lib/python3.11/site-packages/_pytest/fixtures.py\", line 1123, in pytest_fixture_setup\n    result = call_fixture_func(fixturefunc, request, kwargs)\n             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/home/kamaz/PycharmProjects/FinalProject/venv/lib/python3.11/site-packages/_pytest/fixtures.py\", line 895, in call_fixture_func\n    fixture_result = next(generator)\n                     ^^^^^^^^^^^^^^^\n  File \"/home/kamaz/PycharmProjects/FinalProject/conftest.py\", line 26, in browser\n    with sync_playwright() as playwright:\n  File \"/home/kamaz/PycharmProjects/FinalProject/venv/lib/python3.11/site-packages/playwright/sync_api/_context_manager.py\", line 47, in __enter__\n    raise Error(\n"}, "start": 1699988370799, "stop": 1699988370799}, {"name": "browser", "status": "broken", "statusDetails": {"message": "playwright._impl._api_types.Error: It looks like you are using Playwright Sync API inside the asyncio loop.\nPlease use the Async API instead.\n", "trace": "  File \"/home/kamaz/PycharmProjects/FinalProject/venv/lib/python3.11/site-packages/pluggy/_callers.py\", line 77, in _multicall\n    res = hook_impl.function(*args)\n          ^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/home/kamaz/PycharmProjects/FinalProject/venv/lib/python3.11/site-packages/_pytest/fixtures.py\", line 1123, in pytest_fixture_setup\n    result = call_fixture_func(fixturefunc, request, kwargs)\n             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/home/kamaz/PycharmProjects/FinalProject/venv/lib/python3.11/site-packages/_pytest/fixtures.py\", line 895, in call_fixture_func\n    fixture_result = next(generator)\n                     ^^^^^^^^^^^^^^^\n  File \"/home/kamaz/PycharmProjects/FinalProject/conftest.py\", line 26, in browser\n    with sync_playwright() as playwright:\n  File \"/home/kamaz/PycharmProjects/FinalProject/venv/lib/python3.11/site-packages/playwright/sync_api/_context_manager.py\", line 47, in __enter__\n    raise Error(\n"}, "start": 1699988370808, "stop": 1699988370808}, {"name": "browser", "status": "broken", "statusDetails": {"message": "playwright._impl._api_types.Error: It looks like you are using Playwright Sync API inside the asyncio loop.\nPlease use the Async API instead.\n", "trace": "  File \"/home/kamaz/PycharmProjects/FinalProject/venv/lib/python3.11/site-packages/pluggy/_callers.py\", line 77, in _multicall\n    res = hook_impl.function(*args)\n          ^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/home/kamaz/PycharmProjects/FinalProject/venv/lib/python3.11/site-packages/_pytest/fixtures.py\", line 1123, in pytest_fixture_setup\n    result = call_fixture_func(fixturefunc, request, kwargs)\n             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/home/kamaz/PycharmProjects/FinalProject/venv/lib/python3.11/site-packages/_pytest/fixtures.py\", line 895, in call_fixture_func\n    fixture_result = next(generator)\n                     ^^^^^^^^^^^^^^^\n  File \"/home/kamaz/PycharmProjects/FinalProject/conftest.py\", line 26, in browser\n    with sync_playwright() as playwright:\n  File \"/home/kamaz/PycharmProjects/FinalProject/venv/lib/python3.11/site-packages/playwright/sync_api/_context_manager.py\", line 47, in __enter__\n    raise Error(\n"}, "start": 1699988370815, "stop": 1699988370815}, {"name": "browser", "status": "broken", "statusDetails": {"message": "playwright._impl._api_types.Error: It looks like you are using Playwright Sync API inside the asyncio loop.\nPlease use the Async API instead.\n", "trace": "  File \"/home/kamaz/PycharmProjects/FinalProject/venv/lib/python3.11/site-packages/pluggy/_callers.py\", line 77, in _multicall\n    res = hook_impl.function(*args)\n          ^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/home/kamaz/PycharmProjects/FinalProject/venv/lib/python3.11/site-packages/_pytest/fixtures.py\", line 1123, in pytest_fixture_setup\n    result = call_fixture_func(fixturefunc, request, kwargs)\n             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/home/kamaz/PycharmProjects/FinalProject/venv/lib/python3.11/site-packages/_pytest/fixtures.py\", line 895, in call_fixture_func\n    fixture_result = next(generator)\n                     ^^^^^^^^^^^^^^^\n  File \"/home/kamaz/PycharmProjects/FinalProject/conftest.py\", line 26, in browser\n    with sync_playwright() as playwright:\n  File \"/home/kamaz/PycharmProjects/FinalProject/venv/lib/python3.11/site-packages/playwright/sync_api/_context_manager.py\", line 47, in __enter__\n    raise Error(\n"}, "start": 1699988370820, "stop": 1699988370820}, {"name": "browser", "status": "broken", "statusDetails": {"message": "playwright._impl._api_types.Error: It looks like you are using Playwright Sync API inside the asyncio loop.\nPlease use the Async API instead.\n", "trace": "  File \"/home/kamaz/PycharmProjects/FinalProject/venv/lib/python3.11/site-packages/pluggy/_callers.py\", line 77, in _multicall\n    res = hook_impl.function(*args)\n          ^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/home/kamaz/PycharmProjects/FinalProject/venv/lib/python3.11/site-packages/_pytest/fixtures.py\", line 1123, in pytest_fixture_setup\n    result = call_fixture_func(fixturefunc, request, kwargs)\n             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/home/kamaz/PycharmProjects/FinalProject/venv/lib/python3.11/site-packages/_pytest/fixtures.py\", line 895, in call_fixture_func\n    fixture_result = next(generator)\n                     ^^^^^^^^^^^^^^^\n  File \"/home/kamaz/PycharmProjects/FinalProject/conftest.py\", line 26, in browser\n    with sync_playwright() as playwright:\n  File \"/home/kamaz/PycharmProjects/FinalProject/venv/lib/python3.11/site-packages/playwright/sync_api/_context_manager.py\", line 47, in __enter__\n    raise Error(\n"}, "start": 1699988370828, "stop": 1699988370828}, {"name": "browser", "status": "broken", "statusDetails": {"message": "playwright._impl._api_types.Error: It looks like you are using Playwright Sync API inside the asyncio loop.\nPlease use the Async API instead.\n", "trace": "  File \"/home/kamaz/PycharmProjects/FinalProject/venv/lib/python3.11/site-packages/pluggy/_callers.py\", line 77, in _multicall\n    res = hook_impl.function(*args)\n          ^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/home/kamaz/PycharmProjects/FinalProject/venv/lib/python3.11/site-packages/_pytest/fixtures.py\", line 1123, in pytest_fixture_setup\n    result = call_fixture_func(fixturefunc, request, kwargs)\n             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/home/kamaz/PycharmProjects/FinalProject/venv/lib/python3.11/site-packages/_pytest/fixtures.py\", line 895, in call_fixture_func\n    fixture_result = next(generator)\n                     ^^^^^^^^^^^^^^^\n  File \"/home/kamaz/PycharmProjects/FinalProject/conftest.py\", line 26, in browser\n    with sync_playwright() as playwright:\n  File \"/home/kamaz/PycharmProjects/FinalProject/venv/lib/python3.11/site-packages/playwright/sync_api/_context_manager.py\", line 47, in __enter__\n    raise Error(\n"}, "start": 1699988370834, "stop": 1699988370834}], "start": 1699988370834, "stop": 1699988370838}